src/App.jsx

```diff
@@ -1,3 +1,7 @@
+
+import Home from "./pages/Home";
+import AddStudent from "./pages/AddStudent";
+import AllStudents from "./pages/Allstudents";
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import Navbar from './components/Navbar';
 import StudentList from './pages/StudentList';
```

Comment:
Please note that these paths don't exist, hence they cause errors. Work on using the correct paths such as:

```suggestion
import StudentList from "./pages/StudentList";
import AddStudent from "./pages/AddStudent";
```

![image](https://github.com/user-attachments/assets/3d78daac-8a3f-4e89-9dd9-5cfd4252ecdd)